docs/development.rst

```diff
@@ -26,19 +52,52 @@ Quickstart
 
   $ git submodule update --init --recursive
 
+.. - Install the  development :ref:`basic requirements <sec_installation_system_requirements>`.
+.. - Install the Python development requirements using
+..   ``pip install -r python/requirements/development.txt``.
+
+- To work on the Python code, ``cd`` into the ``python`` directory
+- Build the low level module by running ``make`` in the project root.
+- Run the tests to ensure everything has worked: ``python3 -m nose -vs``. These should
+  all pass.
+- Make your changes in a local branch, and open a pull request on GitHub when you
+  are ready. Please make sure that (a) the tests pass before you open the PR; and
+  (b) your code passes PEP8 checks (see below for a git commit hook to ensure this
```

Comment:
this note about PEP8 is maybe out of date - replace with "passes code formatting checks"?